registration-processor/registration-processor-registration-status-service-impl/src/main/java/io/mosip/registration/processor/status/service/impl/InternalAuthDelegateServiceImpl.java

```diff
@@ -82,6 +84,7 @@ public AuthResponseDTO authenticate(AuthRequestDTO authRequestDTO, HttpHeaders h
 		String individualId = getIndividualIdByUserId(authRequestDTO.getIndividualId());
 
 		authRequestDTO.setIndividualId(individualId);
+		authRequestDTO.setIndividualIdType(UIN);
```

Comment:
As per the operator validator code, setIndividualIdType was sent as UIN ?

Reply:
in operator validator, when we are fetching individualType... it is setting individualType to null...

Reply:
if (INDIVIDUAL_TYPE_USERID.equalsIgnoreCase(individualType)) {
			userId = getIndividualIdByUserId(userId);
			individualType = null;
		}

Reply:
Then the same should work here, the internal API has the intellegence to find the type of Id if null is sent.